files/zh-cn/web/api/intersection_observer_api/index.md

```diff
@@ -58,7 +58,7 @@ let observer = new IntersectionObserver(callback, options);
 - `root`
   - : 指定根 (**root**) 元素，用于检查目标的可见性。必须是目标元素的父级元素。如果未指定或者为`null`，则默认为浏览器视窗。
 - `rootMargin`
-  - : 根 (**root**) 元素的外边距。类似于 CSS 中的 {{cssxref("margin")}} 属性，比如 "`10px 20px 30px 40px"` (top, right, bottom, left)。如果有指定 root 参数，则 rootMargin 也可以使用百分比来取值。该属性值是用作 root 元素和 target 发生交集时候的计算交集的区域范围，使用该属性可以控制 root 元素每一边的收缩或者扩张。默认值为 0。
+  - : 根 (**root**) 元素的外边距。类似于 CSS 中的 {{cssxref("margin")}} 属性，比如 "`10px 20px 30px 40px"` (top, right, bottom, left)。如果有指定 root 参数，则 rootMargin 也可以使用百分比来取值。该属性值是用作 root 元素和 target 发生交集时候的计算交集的区域范围，使用该属性可以控制 root 元素每一边的收缩或者扩张。默认值为'0px 0px 0px 0px'。
```

Comment:
```suggestion
  - : 根 (**root**) 元素的外边距。类似于 CSS 中的 {{cssxref("margin")}} 属性，比如 "`10px 20px 30px 40px"` (top, right, bottom, left)。如果有指定 root 参数，则 rootMargin 也可以使用百分比来取值。该属性值是用作 root 元素和 target 发生交集时候的计算交集的区域范围，使用该属性可以控制 root 元素每一边的收缩或者扩张。默认值为四个边距全是 0。
```

![image](https://user-images.githubusercontent.com/15844309/195812160-f2ab5f05-fc68-499c-a7c8-d281ee7ebb8e.png)

Reply:
> ```suggestion
> 
>   - : 根 (**root**) 元素的外邊距。類似於 CSS 中的 {{cssxref(“margin”)}} 屬性，比如 “`10px 20px 30px 40px”` (top, right, bottom, left)。如果有指定 root 參數，則 rootMargin 也可以使用百分比來取值。該屬性值是用作 root 元素和 target 發生交集時候的計算交集的區域範圍，使用該屬性可以控制 root 元素每一邊的收縮或者擴張。默認值為四個邊距全是 0。
> 
> ```
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/15844309/195812160-f2ab5f05-fc68-499c-a7c8-d281ee7ebb8e.png)
> 
> 

Although your suggestion is closer to the original English document, but it weakens the unit description.

https://www.w3.org/TR/intersection-observer/ 
According to specification,default rootMargin is “0px 0px 0px 0px ”